0.Preprocess_Data/3.data_aggregation.ipynb

```diff
@@ -10,7 +10,7 @@
         },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #63.    elif aggregation == False and nomic == False:

</pre><p>Instead of adding lines 63 to 66, and the other checks, consider checks, consider checking if these variables are boolean at the beginning of the file.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>